applications/serial_lte_modem/src/slm_at_commands.c

```diff
@@ -228,52 +235,55 @@ static int handle_at_slmuart(enum at_cmd_type type)
 	int ret = -EINVAL;
 
 	if (type == AT_CMD_TYPE_SET_COMMAND) {
-		uint32_t baudrate = 115200;
-
-		if (at_params_valid_count_get(&at_param_list) > 1) {
-			ret = at_params_unsigned_int_get(&at_param_list, 1, &baudrate);
-			if (ret) {
-				LOG_ERR("AT parameter error");
+		uint32_t baudrate = slm_uart.baudrate;
+		uint16_t hwfc = slm_uart.flow_ctrl;
```

Comment:
Do you really need to initialize here? The actual value of `baudrate` and `hwfc` is only used if obtained successfully from `at_param_list`.

Reply:
Trying to be compatible with original command behavior.
Original would apply default baudrate (115200) even if there is no parameter obtained.

Reply:
Yes, but the previous version would actually make use of the `baudrate` if it failed to read the value from the command. Now, the `baudrate` is only processed below if successfully overwritten.